webui/react/src/components/ProjectCard.tsx

```diff
@@ -41,49 +43,56 @@ const ProjectCard: React.FC<Props> = ({
 
   const classnames = [css.base];
   if (project.archived) classnames.push(css.archived);
+  if (project.workspaceId === 1) classnames.push(css.uncategorized);
 
   return (
     <Card
       actionMenu={!project.immutable && !hideActionMenu ? menu : undefined}
       onClick={(e: AnyMouseEvent) => handlePath(e, { path: paths.projectDetails(project.id) })}
       onDropdown={onClick}>
       <div className={classnames.join(' ')}>
-        <div className={css.headerContainer}>
-          <Typography.Title className={css.name} ellipsis={{ rows: 3, tooltip: true }} level={5}>
-            {project.name}
-          </Typography.Title>
-        </div>
-        <div className={css.workspaceContainer}>
-          {showWorkspace && project.workspaceId !== 1 && (
-            <Tooltip content={project.workspaceName}>
-              <div className={css.workspaceIcon}>
-                <Avatar palette="muted" size={Size.Small} square text={project.workspaceName} />
+        <Column>
+          <Row justifyContent="space-between" width={125}>
+            <Title size={TypographySize.XS} truncate={{ rows: 1, tooltip: true }}>
+              {project.name}
+            </Title>
+          </Row>
+          <Row>
+            <div className={css.workspaceContainer}>
+              {showWorkspace && (
+                <Tooltip content={project.workspaceName}>
```

Comment:
`Avatar` already has a tooltip, should remove this redundant `Tooltip`

Reply:
✅